src/main/java/seedu/duke/ui/AppUi.java

```diff
@@ -113,6 +114,17 @@ public void printHelpMenu() {
         helpCommand.execute(financialList);
     }
 
+    /**
+     * Executes the exit command and returns the status of the application.
+     * 
+     * @return {@code true} if the application should continue running, {@code false} if it should exit.
+     */
+    public boolean exitCommand() {
+        ExitCommand exitCommand = new ExitCommand();
+        exitCommand.execute(financialList);
+        return !exitCommand.isExit();
```

Comment:
Would it be less awkward if the method was called `shouldContinueLoop()` , so that the statement here would be `return shouldContinueLoop()` (i.e. no need to use the ! to invert the output)

Reply:
True, modified in next commit